src/scss/sections/_hero-banner.scss

```diff
@@ -0,0 +1,29 @@
+.hero-carousel {
+  .carousel-indicators {
+    position: absolute;
+    left: 0;
+    bottom: calc-em(20);
+    margin: 0 auto;
+    padding: 0 calc-em(11);
```

Comment:
please apply these changes to the main carousel component using the variables, this is the unique carousel where we use indicators (except the sidebar carousel on blog mobile that we should check before/after the update)